tests/test_issue.py

```diff
@@ -16,13 +17,14 @@ def test_issue_from_issue_number(test_repo, issue_json, httpx_mock):
         json=issue_json,
     )
 
-    with httpx.Client() as _: 
+    with httpx.Client() as _:
         issue = Issue.from_issue_number(repo=test_repo, issue_id=14)
```

Comment:
This should also be updated to ensure the tests are referencing the right variable name:
```suggestion
        issue = Issue.from_issue_number(repo=test_repo, issue_number=14)
```